.github/workflows/self-hosted-gpu-test.yml

```diff
@@ -66,7 +66,7 @@ jobs:
       - name: Refine test env
         shell: bash -l {0}
         run: |
-          mamba install -y cudatoolkit==11.0.3 openmm==7.6
+          mamba install -y cudatoolkit==11.0.3 openmm==7.7
```

Comment:
I wonder if not pinning any version should be enough to always get the latest supported version.

Reply:
In theory it should, I like to use this sometimes as a canary though, like if something happens to the conda-forge recipe that makes 7.7 not able to install, and so we grab 7.6, I'd rather get an error saying the env couldn't solve then just test on 7.6. Thoughts? If we got some aws credits, I'd do a more proper matrix which would grab versions we want + nightly.

Reply:
Yeah, if we have the credits I guess a matrix would be better. I was also thinking that we probably want to match what we have in our CI test environment [here](https://github.com/choderalab/perses/blob/main/devtools/conda-envs/test_env.yaml#L20), I think that would actually make more sense to correlate GHA CI tests with the AWS GPU tests, if needed.

Reply:
I think you're right! I'll specify the version like that, good idea!